storage/ndb/rest-server/data-access-rondb/src/rdrs-hopsworks-dal.cpp

```diff
@@ -120,7 +120,7 @@ RS_Status find_api_key_int(Ndb *ndb_object, const char *prefix, HopsworksAPIKey
     return RS_CLIENT_ERROR("Wrong length of the search key");
   }
 
-  char cmp_str[col_size];
+  char* cmp_str = new char[col_size];
```

Comment:
memory leak. in the schema.go file you will find the table definitions for hopsworks database. create an appropriate constant

Reply:
Can you please do this? I already fixed all others except the ones in this file.